src/browser/new-browser.ts

```diff
@@ -226,13 +225,26 @@ export class NewBrowser extends Browser {
         config: BrowserConfig,
         capabilities: WebdriverIO.Capabilities,
     ): Promise<WebdriverIO.Capabilities> {
-        const browserNameLowerCase = config.desiredCapabilities?.browserName?.toLowerCase() as string;
+        const { getNormalizedBrowserName, installBrowser } = await import("../browser-installer");
+        const normalizedBrowserName = getNormalizedBrowserName(this._config.desiredCapabilities?.browserName);
+
+        if (!normalizedBrowserName) {
+            throw new Error(
+                [
+                    `Running auto local "${this._config.desiredCapabilities?.browserName}" is unsupported`,
+                    `Supported browsers: "chrome", "firefox", "safari", "edge"`,
+                ].join("\n"),
+            );
+        }
+
         const executablePath = await installBrowser(
-            this._config.desiredCapabilities?.browserName as SupportedBrowser,
-            this._config.desiredCapabilities?.browserVersion as string,
+            normalizedBrowserName,
+            this._config.desiredCapabilities?.browserVersion,
+            { shouldInstallWebDriver: false, shouldInstallUbuntuPackages: true },
```

Comment:
Explicitly setting "should install" options, as they are all set to "false" now by default.

In "_addExecutablePath" we dont need to install webdriver as user might want to run browser in "devtools" protocol